presto-main/src/test/java/com/facebook/presto/type/TestTypeRegistry.java

```diff
@@ -194,14 +194,16 @@ public void testTypeCompatibility()
         assertThat("row(varchar(2))", "row(varchar(5))").hasCommonSuperType("row(varchar(5))").canCoerceFirstToSecondOnly();
 
         assertThat("row(a integer)", "row(a bigint)").hasCommonSuperType("row(a bigint)").canCoerceFirstToSecondOnly();
-        assertThat("row(a integer)", "row(b bigint)").hasCommonSuperType("row(bigint)").cannotCoerceToEachOther();
-        assertThat("row(integer)", "row(b bigint)").hasCommonSuperType("row(bigint)").cannotCoerceToEachOther();
+        assertThat("row(a integer)", "row(b bigint)").hasCommonSuperType("row(bigint)").canCoerceFirstToSecondOnly();
```

Comment:
Why canCoerceFirstToSecondOnly? Isn't the point for these to be coercible to each other?

Reply:
one is integer, the other is bigint.

Reply:
Doh! I was so focused on the names that I ignored the field types.